timescaledb/how-to-guides/continuous-aggregates/real-time-aggregates.md

```diff
@@ -28,4 +28,94 @@ You can enable and disable real time aggregation by setting the
 
 </procedure>
 
+## Limitations
```

Comment:
IMO, this example is overly long and I am not sure the use will be able to understand exactly what the take-away is. 

I also think we need to update the description of real-time aggregates above. The misleading description needs to be fixed. In particular this:

```
Real time aggregates use the aggregated data and add the
most recent raw data to it to provide accurate and up to date results, without
needing to aggregate data as it is being written.
```

Reply:
This description seems accurate 🤷‍♂️  Speaking of the example, I don't see a way to make it shorter, but feel free to suggest an alternative code. The take-aways are in the text, so the user can just skip the example.

Reply:
I agree with Erik.  This section describes the problem at length but we don't describe any steps on how to mitigate the problem. The emphasis should be on how to get expected behavior (setting up policies appropriately) +  gotchas (manual refresh calls that can interfere with the materialized boundaries) and use the example within this context.